secret_service/secret_service.go

```diff
@@ -97,7 +97,13 @@ func (s *SecretService) GetCollection(name string) dbus.BusObject {
 func (s *SecretService) GetLoginCollection() dbus.BusObject {
 	path := dbus.ObjectPath(collectionBasePath + "login")
 	if err := s.CheckCollectionPath(path); err != nil {
-		path = dbus.ObjectPath(loginCollectionAlias)
+		var loginAlias dbus.ObjectPath
+		err := s.object.Call(serviceInterface+".ReadAlias", 0, "login").Store(&loginAlias)
+		if err != nil || loginAlias == "/" {
+			path = dbus.ObjectPath(loginCollectionAlias)
```

Comment:
If the PR is about adding a fallback I would assume that the current behavior stays the same and the new thing is the fallback. Here we have it the other way around.

Maybe it's good, I don't know the details and give the impact for >1000 projects that use this library, I would like to have a reason why we change the order.